src/components/card/ContentCard.tsx

```diff
@@ -0,0 +1,43 @@
+import Image from 'next/image';
+import React from 'react';
+
+export type ContentCardProps = {
+  imgSrc?: string;
+  title: string;
+  description: string;
+  tag?: string;
+};
+
+const ContentCard = ({ imgSrc = '/images/contentCardDefault.png', title, description, tag }: ContentCardProps) => {
+  return (
+    <div 
+      className="
+      w-72 max-w-[281px] h-auto min-h-[387px] md:w-[28rem] md:max-w-[450px] md:min-h-[477px] 
```

Comment:
One suggestion I have is to standardize the width and height units. Because sometimes it's used "px" and sometimes it's used "rem".

Also, the base class "w-72" (which would convert to 288px) is being overridden by the "max-w-[281px]" class. But I believe "max-width" and "max-height" properties are most appropriate when the width and height are flexible, which is not the case here.

What do you think?

Reply:
Great tips, thank you for the information!